deploy/operator/10_pdb.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: scylla-operator
+  namespace: scylla-operator
+spec:
+  minAvailable: 1
```

Comment:
nit: `maxUnavailable: 1` is a bit more flexible - if a user deploys 3 replicas to stay HA even during drain, e.g. for random machine failure, it would still hold true.